modules/images-configuration-parameters.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration.adoc
+
+[id="images-configuration-parameters_{context}"]
+= Image controller configuration parameters
+
+The `image.config.openshift.io/cluster` resource offers the following
+configuration parameters.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`metadata`
+|Standard object's metadata.
+
+|`spec`
+|Holds user-settable values for configuration.
+
+|`status`
+|Holds observed values from the cluster.
+
+|`allowedRegistriesForImport`
+|Limits the container image registries from which normal users may import images.
+Set this list to the registries that you trust to contain valid Docker images,
+and that you want applications to be able to import from. Users with permission
+to create images or `ImageStreamMappings` from the API are not affected by this
+policy. Typically only cluster administrators will have the appropriate permissions.
+
+|`additionalTrustedCA`
+|A reference to a ConfigMap containing additional CAs that should be trusted
+during ImageStream import, pod image pull, and `openshift-image-registry`
+pullthrough.
+
+The namespace for this ConfigMap is `openshift-config`.
+
+|`registrySources`
+|Contains configuration that determines how the container runtime should treat
+individual registries when accessing images for build sand pods. For instance,
```

Comment:
s/build sand pods/builds and pods/
(and if that was copied from our api doc then it needs to be fixed there too)

Reply:
That was just my typo. :/ It's "builds+pods" in the API docs.